src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/CourtTransfersResource.kt

```diff
@@ -0,0 +1,90 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.resources
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.welcometoprison.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.CourtTransferInDetail
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.TransferIn
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.TransfersService
+import javax.validation.Valid
+import javax.validation.constraints.NotEmpty
+import javax.validation.constraints.NotNull
+
+@RestController
+@Validated
+@RequestMapping(name = "Court Transfers", produces = [MediaType.APPLICATION_JSON_VALUE])
+class CourtTransfersResource(
```

Comment:
I'm not sure we need a new endpoint here... we were trying to keep the frontend agnostic to the type of return.

Reply:
will use the arrivals endpoint